esx_service/utils/vmdk_utils.py

```diff
@@ -64,6 +64,9 @@
 VMDK_RETRY_COUNT = 5
 VMDK_RETRY_SLEEP = 1
 
+# root for all the volumes
+VOLUME_ROOT = "/vmfs/volumes/"
```

Comment:
this string has been used as a part regex in some. Will do this as a part refactoring all constants in our vmdkops and vmdk_utils separately.